src/CONST.ts

```diff
@@ -1552,6 +1552,10 @@ const CONST = {
             CATEGORIZE: 'categorize',
             SHARE: 'share',
         },
+        ACTION_TYPE: {
```

Comment:
@nkdengineer  IMO, we don't need to create a new type here. Let's reuse REPORT_ACTION_TYPE and create a new type PreviewPaymentMethodType like this

```
type PreviewPaymentMethodType = DeepValueOf<typeof CONST.IOU.REPORT_ACTION_TYPE.PAY | typeof CONST.IOU.REPORT_ACTION_TYPE.APPROVE>;

```

Reply:
@DylanDylann I updated.